### ISSUE TITLE
Exporting 'enum' as 'type' makes it unusable

### ISSUE THREAD

**Issue by CHE1RON** [kubb-labs/kubb#1363]
### What version of `kubb` is running?

3.0.0-beta.9

### What platform is your computer?

macOS

### What version of external packages are you using(`@tanstack-query`, `MSW`, `React`, `Vue`, ...)

3.0.0-beta.9

### What steps can reproduce the bug?

Config:

```typescript
pluginTs({
  output: { path: 'types' },
  enumType: 'enum',
}),
```

The generated barrel files export everything (including `enum` objects) as `type`, which leads to the following error when using them:

<img width="808" alt="Screenshot 2024-10-30 at 21 39 20" src="https://github.com/user-attachments/assets/bf7c5b24-20f3-424b-993a-a0ff291482f3">

.. so I can only use them when importing directly from its file 🥴 

### How often does this bug happen?

Every time

### What is the expected behavior?

`enum`s are not exported as `type`s

**Comment by stijnvanhulle** [comment]
@CHE1RON This has been fixed with v`3.0.0-beta.11`, we were using `export type` instead of `export`.

**Comment by CHE1RON** [comment]
That's very good to hear, and just shows how aswesome this library is!
